cplusplus/Wawenet/helpScreen.txt

```diff
@@ -28,14 +30,18 @@ L=1: normalization on (Default)
 
 -- use option -s S to specify segment step (stride).
 WAWEnet requires a full 3 seconds of signal to generate a result. 
-If a .wav file is longer than 3 seconds multiple results may be produced.
-S specifies the number of samples to move ahead in the file when
-extracting the next segment. The default value of S is 48,000. This gives
-zero overlap between segments and any speech file less than 6 sec.
-will produce one result, based on just the first 3 sec. A 6 sec.
-speech file will produce two results.
-If S = 24,000 for example, segment overlap will be 50%, a 4.5 sec.
-file will produce 2 results and a 6 sec. file will produce 3 results.
+If the input speech .wav file is longer than 3 seconds
+multiple results may be produced. S specifies the number of samples to
+move ahead in the file when extracting the next segment.
+The default value of 48,000 gives zero overlap between
+segments. Using this default any input less than 6 sec. will produce one
+result, based on just the first 3 sec. A 6 sec. input will produce two
+results. If S = 24,000 for example, segment overlap will be 50%, a 
```

Comment:
Probably ultimately a Lilli question, but my two cents:

Using the shortened version of seconds `sec.` here seems a bit ambiguous, particularly with the other case of `3 sec.` which has the `.` actually function as a full stop for the sentence as opposed to indicating the shortened version of seconds. Maybe it'd be better to use `sec` or `seconds` for this text block, then the `.` isn't ambiguous?